### DISCUSSION TITLE
No sysmon logs in Standalone 2.3.110

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#7493] [Unsupported Versions]
Updated my Standalone Security Onion instance to 2.3.110 last night about 6 pm. Agent version on endpoints is 7.16.0.

After update I noticed that sysmon logs were not coming in. I deployed agent version 7.17.1 to endpoint and verified it was running with a stable connection to SO2 SIEM on port 5044. Second screenshot.

Is this an index issue, ECS issue, or perhaps something else? Still no sysmon logs....

![image](https://user-images.githubusercontent.com/70167373/157668401-2902ad2d-c500-4074-b9c3-9e909b51f568.png)

![image](https://user-images.githubusercontent.com/70167373/157668470-7026d1ff-d19b-43a1-aa35-4fd8498b2adb.png)

Update: I redeployed and added Windows Defender to be logged as well. Defender logs came back successfully but still no sysmon.

![image](https://user-images.githubusercontent.com/70167373/157674433-9fedba96-1dc5-4b13-9351-fd3203aea6c3.png)

**Comment by weslambert** [comment]
A few things to check:

- Is the WLB version the OSS version?

- Are there any error messages from WLB if you put it in debug mode?

- Are there any errors in the Logstash log on the search node(s)?

- Is the actual Elasticsearch index is growing for Beats?

`sudo so-elasticsearch-query _cat/indices | grep beats | sort -n`

- Has the index rolled over to the next day since upgrading ?

**Comment by ColeVan** [comment]
I deleted and reload indexes about an hour ago. The beat index is growing.

Went to endpoint Win-10-04 to verify sysmon is collecting, connection to SO2 on port 5044, and ran debug for winlogbeat. I don't see any major issues. The WLB version is 7.17.1.

![image](https://user-images.githubusercontent.com/70167373/157683552-13ff2fbf-f300-4eba-81f4-54cadbbc0ddd.png)
![image](https://user-images.githubusercontent.com/70167373/157683620-507ff119-3e28-46d3-8716-838706ffbeda.png)

**Reply by ColeVan** [reply]
I just checked The Hunt Tab and I see sysmon logs there but not in Kibana. 
![image](https://user-images.githubusercontent.com/70167373/157684228-cb99a5b7-8fd1-4ad6-8f12-f3cdc0c3767c.png)
![image](https://user-images.githubusercontent.com/70167373/157684359-63c4d3d8-c6a9-4865-9388-f468a1606fc2.png)

**Comment by weslambert** [comment]
Will you please post a screenshot of a search for `event.category` and `event.module` in `Kibana -> Index Patterns`?

**Reply by ColeVan** [reply]
It's there. 

So interestingly enough the Process CLI dashboard under the main Host dashboard may be the issue. When I edit the Process CLI visualization individually I see the logs but not under main Host Dashboard.

![image](https://user-images.githubusercontent.com/70167373/157685075-a22f0ef0-1e1f-4c4f-bb07-a1d2d1636f5d.png)

![image](https://user-images.githubusercontent.com/70167373/157685805-3f37d1e6-c40d-44c6-907d-33f4f1b86799.png)

![image](https://user-images.githubusercontent.com/70167373/157685993-c6cba05b-8596-41eb-a492-acb8d6c6ff6e.png)

**Comment by weslambert** [comment]
What does `process.command_line` look like in the index patterns?

**Reply by weslambert** [reply]
Also, what are you editing the process CLI visualization to use?

**Comment by ColeVan** [comment]
process.command_line looks good.  It seems to me that the preset for Host Dashboard "event.category:host" is the problem. I took it off and it works fine. Maybe a new preset should go there instead of event.category:host
![image](https://user-images.githubusercontent.com/70167373/157687884-edccd776-2ae6-4e15-a071-f899b8625b12.png)
![image](https://user-images.githubusercontent.com/70167373/157689787-e3b7bcca-9e5a-4104-b603-7448ed9a0822.png)
![image](https://user-images.githubusercontent.com/70167373/157690044-ab8bfe1b-0553-4ce7-882c-8601c7e667fb.png)

**Comment by weslambert** [comment]
Yes, this is why I was asking for screenshots of event.category and event.module before 😄  If there are conflicts there, then you'll have to use event.category.keyword. However, in this case it seems like it's just missing (if I'm reading it correctly, and it's not displaying as such because of the conflict).

**Reply by ColeVan** [reply]
Hahaha... my bad. I need to learn to read better. I went around the world to find the problem when all along it was just one word lol

**Reply by weslambert** [reply]
To confirm, are you using the OSS version of WLB?

**Reply by ColeVan** [reply]
Sorry.. what is OSS stand for again?

 I'm using version between 7.16 and 7.17.1

**Reply by weslambert** [reply]
https://www.elastic.co/downloads/past-releases/winlogbeat-oss-7-17-1

**Reply by ColeVan** [reply]
Ahh yes. I am indeed using OSS 7.16 and 7.17.1

**Reply by weslambert** [reply]
What does the mapping for `event.category` in Index Patterns look like?
